### PR TITLE
Create a read-only user

### PR DESCRIPTION
**Pull request by danieltprice**
Added the following topics under Manage > Access:
- Manage project access (project-level access)
https://neon-next-git-docs-read-only-user-neondatabase.vercel.app/docs/manage/project-access
- Manage database access (covers creating read-only users, read-write users, developer users in a branching context)
https://neon-next-git-docs-read-only-user-neondatabase.vercel.app/docs/manage/database-access

### CODE REVIEW

**Review by atumark — APPROVED** [review]

File: content/docs/manage/database-access.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+subtitle: Learn how to manage user access to databases in your Neon project
+enableTableOfContents: true
+redirectFrom:
+  - /docs/guides/manage-database-access
+updatedOn: '2023-09-15T13:00:43Z'
+---
+
+Each Neon project is created with a default Postgres role that takes its name from your Neon account (the account you registered with). For example, if a user named "Alex" signs up for Neon, the project is created with a default role named `alex`.
+
+The default Postgres role is automatically assigned the [neon_superuser](/docs/manage/roles#the-neonsuperuser-role) role, which allows creating databases, roles, and reading and writing data in all tables, views, sequences. Any user created with the Neon console, Neon API, or Neon CLI is also assigned the `neon_superuser` role.
```
Comment (bgrenon) [inline content/docs/manage/database-access.md:12]:
```suggestion
The default Postgres role is automatically assigned the [neon_superuser](/docs/manage/roles#the-neonsuperuser-role) role, which allows creating databases, roles, and reading and writing data in all tables, views, and sequences. Any user created with the Neon console, Neon API, or Neon CLI is also assigned the `neon_superuser` role.
```

File: content/docs/manage/database-access.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    CREATE ROLE readonly_user1 WITH LOGIN PASSWORD '<password>';
+    ```
+
+5. Grant the user membership in the `readonly` role:
+
+    ```sql
+    GRANT readonly TO readonly_user1;
+    ```
+
+    The `readonly_user1` user now has read-only access to tables in the specified schema and database and should able to connect and run `SELECT` queries.
```
Comment (bgrenon) [inline content/docs/manage/database-access.md:89]:
```suggestion
    The `readonly_user1` user now has read-only access to tables in the specified schema and database and should be able to connect and run `SELECT` queries.
```

File: content/docs/reference/glossary.md [hunk 3]
```diff
@@ -265,11 +265,9 @@ An open-source relational database management system (RDBMS) emphasizing extensi
 
 ### Postgres role
 
-A Postgre role named for the registered Neon account is created with each Neon project. This user can access a Neon database from a client. Older projects may have a `web-access` system role, used by the [SQL Editor](#sql-editor) and Neon’s [Passwordless auth](#passwordless-auth). The `web-access` role is system managed. It cannot be modified, removed, or used in other authentication scenarios.
+A Postgres role named for the registered Neon account is created with each Neon project. This role and any additional role created in the Neon Console, API, or CLI is assigned the [neon_superuser](/docs/manage/roles#the-neonsuperuser-role) role, which allows creating databases, roles, and reading and writing data in all tables, views, sequences. Roles created with SQL are created with the same basic [public schema privileges](/docs/manage/database-access#public-schema-privileges) granted to newly created roles in a standalone Postgres installation. These users are not assigned the `neon_superuser` role. They must be selectively granted permissions for each database object. For more information, see [Manage database access](/docs/manage/database-acess).
```
Comment (bgrenon) [inline content/docs/reference/glossary.md:268]:
```suggestion
A Postgres role named for the registered Neon account is created with each Neon project. This role &#8212; and any additional roles created in the Neon Console, API, or CLI) is assigned the [neon_superuser](/docs/manage/roles#the-neonsuperuser-role &#8212; role, which allows creating databases, roles, and reading and writing data in all tables, views, sequences. Roles created with SQL are created with the same basic [public schema privileges](/docs/manage/database-access#public-schema-privileges) granted to newly created roles in a standalone Postgres installation. These users are not assigned the `neon_superuser` role. They must be selectively granted permissions for each database object. For more information, see [Manage database access](/docs/manage/database-acess).
```

**Comment by bgrenon** [comment]
Solid! A few minor suggested edits.

**Review by bgrenon — APPROVED** [review]

File: content/docs/manage/database-access.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+
+1. Connect to your database from an SQL client such as [psql](/docs/connect/query-with-psql-editor), [pgAdmin](https://www.pgadmin.org/), or the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor). If you need help connecting, see [Connect from any client](/docs/connect/connect-from-any-app).
+
+2. Create a `readonly` role using the following statement:
+
+    ```sql
+    CREATE ROLE readonly PASSWORD '<password>';
+    ```
+  
+    Neon requires specifying a password. The password must have 60 bits of entropy (at least 12 characters with a mix of lowercase, uppercase, number, and symbol characters). For detailed password guidelines, see [Manage roles with SQL](/docs/manage/roles#manage-roles-with-sql).
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/manage/database-access.md]: Good point. Removed.

**Pull request merged**